DistributedLock.Tests/Infrastructure/TestingSynchronizationStrategy.cs

```diff
@@ -21,5 +21,6 @@ public abstract class TestingSynchronizationStrategy : IDisposable
         public virtual IDisposable? PrepareForHandleLost() => null;
         public virtual void PrepareForHighContention() { }
         public virtual void Dispose() { }
+        public virtual string? GetLockPrefix() => null;
```

Comment:
Why is this needed?

Reply:
The need to run RunLockTaker / DistributedLockTaker with global lock name (prefix + local name) in order to pass cross process tests.